markdown/Carbon/URLQuery.swift

```diff
@@ -0,0 +1,25 @@
+//  Copyright © 2019 The nef Authors.
+
+import Foundation
+
+// MARK: Constants
+extension URLRequest {
+    static let URLLenghtAllowed = 3000
```

Comment:
I would use `urlLengthAllowed` instead, to match https://developer.apple.com/documentation/foundation/characterset/1780127-urlpathallowed

Reply:
Done!